### ISSUE TITLE
Missing WASM PlutusScriptWitness constructor

### ISSUE THREAD

**Issue by solidsnakedev** [dcSpark/cardano-multiplatform-lib#297]
CML 4.0.2
There's no way to instantiate a `PartialPlutusWitness` from `PlutusScriptWitness`, constructor is missing

```javascript
class PlutusScriptWitness {

    static __wrap(ptr) {
        const obj = Object.create(PlutusScriptWitness.prototype);
        obj.ptr = ptr;
        PlutusScriptWitnessFinalization.register(obj, obj.ptr, obj);
        return obj;
    }

    __destroy_into_raw() {
        const ptr = this.ptr;
        this.ptr = 0;
        PlutusScriptWitnessFinalization.unregister(this);
        return ptr;
    }

    free() {
        const ptr = this.__destroy_into_raw();
        wasm.__wbg_plutusscriptwitness_free(ptr);
    }
    /**
    * @returns {ScriptHash}
    */
    hash() {
        const ret = wasm.plutusscriptwitness_hash(this.ptr);
        return ScriptHash.__wrap(ret);
    }
}
module.exports.PlutusScriptWitness = PlutusScriptWitness;
```

**Comment by rooooooooob** [comment]
@solidsnakedev what about `PartialPlutusWitness.new(script: PlutusScriptWitness, data: PlutusData)`?

```js
export class PartialPlutusWitness {
  free(): void;
/**
* @param {PlutusScriptWitness} script
* @param {PlutusData} data
* @returns {PartialPlutusWitness}
*/
  static new(script: PlutusScriptWitness, data: PlutusData): PartialPlutusWitness;
/**
* @returns {PlutusScriptWitness}
*/
  script(): PlutusScriptWitness;
/**
* @returns {PlutusData}
*/
  data(): PlutusData;
}
```

**Comment by solidsnakedev** [comment]
right but , how do I construct a `PlutusScriptWitness` class?

**Comment by rooooooooob** [comment]
Ah sorry I misunderstood. I guess we missed it in the WASM ones since on the rust side it's via the `From` traits not as a method.

PR that resolves this: #305 (also a couple other missing script-related API I noticed).

Please let us know if you find anything else missing! Thanks for the report.

**Comment by solidsnakedev** [comment]
thank you so much @rooooooooob :)

**Comment by rooooooooob** [comment]
@solidsnakedev just fyi we released 5.1.0 a new version that includes this missing API
